### ISSUE TITLE
Add Appropriate content from the content doc to website at appropriate places

### ISSUE THREAD

**Issue by strongSoda** [UdacityFrontEndScholarship/jeevan-rakht#226]
All the content is available in the doc hosted in repo. Your task is to add that content in appropriate divs under appropriate headings in the website homepage.

The current site is here https://jeevanrakht.herokuapp.com/index.html

**Comment by strongSoda** [comment]
@karthik781

Can you pls elaborate how much and by when will u complete this task to give us a clear idea?

**Comment by Karthik781** [comment]
@strongSoda  By end of tomorrow. aren't pages are connected to homepage yet?

**Comment by gauravsinghaec** [comment]
@Karthik781 please let us know if you are still working on it?

**Comment by gauravsinghaec** [comment]
@Karthik781 any update? are you facing any roadblocks?

**Comment by Karthik781** [comment]
@gauravsinghaec Sorry man, last week was hectic with internship interviews etc.. can you give me this weak?

**Comment by gauravsinghaec** [comment]
hey @Karthik781 that's totally ok. Yeah, I can understand but it will help the team if you can give us the clear idea. Sometimes honest feedback helps so let me ask specifically, 
1. Did you start the work?
 If yes then what percentage of tasks are done until today?
2. Will you be comfortable or want us to pass it to other if you are not getting enough time.?

**Comment by Karthik781** [comment]
@gauravsinghaec  Didn't started yet! will complete by weekend if thats okay.

**Comment by Ruchika30** [comment]
@gauravsinghaec  Can I work on it?

**Comment by gauravsinghaec** [comment]
@Ruchika30 yeah sure, before that we would like to know if you can come up with some work before the weekend.

**Comment by Ruchika30** [comment]
@gauravsinghaec I went though the content of the doc & the site too. Please tell me am I suppose to insert the complete information provided in the doc on the page ? The content is highly informative.

**Comment by gauravsinghaec** [comment]
@Ruchika30 that's so nice of you. Please work on the homepage related content first and share your work here. Then you can work on other content. is that ok?

**Comment by Ruchika30** [comment]
@gauravsinghaec Sorry to say , but it looked a mess after the whole detailed content being inserted when I just tried inserting for "Why donate". I am following the doc given in the above link.  https://github.com/UdacityFrontEndScholarship/jeevan-rakht/blob/master/_workables/content
Pls suggest.

**Comment by gauravsinghaec** [comment]
@Ruchika30 as mentioned
The content is in _workabkes/content folder here https://github.com/UdacityFrontEndScholarship/jeevan-rakht/blob/master/_workables/content/JeevanRakht%20-%20Content%20Doc%20(final).pdf

The current site is here https://jeevanrakht.herokuapp.com/index.html
You can see the myth section is blank on homepage (Open the site from above link)

**Comment by gauravsinghaec** [comment]
@samyak8853 Great show buddy. Can you please deploy it on Heroku or share any URL to see the changes live? 
Few things to note:
1. The **&** in navigation (Faq & info) is not corrected by HTML utility **&amp;**
2. I would check if we need to have two separate page for Faq and Info or it will be a single page.
3. We have **main.css** for website, is there any specific reason for creating another css file **styles.css**

**Comment by codestryker** [comment]
Today i will clear the issues for my pull request.

**Comment by gauravsinghaec** [comment]
All good so far @samyak8853. Can you please reduce your commits before I merge your PR. Now you can see there are 40 commits. Also please give some meaningful comments to your commit messages. Please follow the below link for the good git commit messages.
https://udacity.github.io/git-styleguide/
